packages/editor/src/index.ts

```diff
@@ -1,5 +1,5 @@
 import { AsyncResource } from 'node:async_hooks';
-import { editAsync } from 'external-editor';
+import { editAsync, IFileOptions } from 'external-editor';
```

Comment:
Can you add documentation of this new option in the prompt README?

Reply:
I added the docs for both `editor readme file`  and `global docs`